src/Nncase.Tests/Evaluator/UnitTestEvaluatorMath.cs

```diff
@@ -0,0 +1,1033 @@
+// Copyright (c) Canaan Inc. All rights reserved.
+// Licensed under the Apache license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.IO;
+using System.Linq;
+using Autofac;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using NetFabric.Hyperlinq;
+using Nncase.Evaluator;
+using Nncase.IR;
+using Nncase.IR.F;
+using Nncase.IR.Math;
+using Nncase.IR.Tensors;
+using Nncase.TestFixture;
+using Nncase.Utilities;
+using OrtKISharp;
+using Xunit;
+using static Nncase.IR.F.Math;
+using static Nncase.IR.F.NN;
+using static Nncase.IR.F.Tensors;
+using static Nncase.Utilities.DumpUtility;
+using RangeOf = Nncase.IR.Math.RangeOf;
+using Tuple = Nncase.IR.Tuple;
+
+namespace Nncase.Tests.EvaluatorTest;
+
+public class UnitTestEvaluatorMath : TestFixture.UnitTestFixtrue
+{
+    [Fact]
+    public void TestBinaryScalarScalar()
+    {
+        var ops = new BinaryOp[]
+        {
+            BinaryOp.Add, BinaryOp.Sub, BinaryOp.Mul, BinaryOp.Div, BinaryOp.Mod, BinaryOp.Min, BinaryOp.Max, BinaryOp.Pow,
+            BinaryOp.LogicalAnd, BinaryOp.LogicalOr, BinaryOp.LogicalXor, BinaryOp.LeftShift, BinaryOp.RightShift,
+        };
+
+        // bool
+        foreach (var op in ops)
```

Comment:
You can use MemberData to build the input for the whole test, instead of putting the input inside the test body, so that you can quickly locate which set of parameters is wrong when something goes wrong. You can refer to UnitTestCombineTranspose.cs for this.

Reply:
这个还看不太懂， 后面再整复杂的吧